src/queue.py

```diff
@@ -0,0 +1,31 @@
+from dll import DoublyLinkedList as DLL
+
+
+class Queue(object):
+    """Python Queue Class."""
+
+    def __init__(self, iterable=None):
+        """Initialize Queue class."""
+        self.dll = DLL(iterable)
```

Comment:
It would be more in line with Pyothn convention to prepend an underscore onto this attribute, which is regarded as private to the user, i.e. `self._dll`